cms/serializers.py

```diff
@@ -39,10 +45,54 @@ def get_financial_assistance_form_url(self, instance):
             else ""
         )
 
+    def get_next_run_id(self, instance):
+        """Get next run id"""
+        run = instance.course.first_unexpired_run
+        return run.id if run is not None else None
```

Comment:
We can simplify the condition like:
```suggestion
        return run.id if run else None
```